lib/side_menu_animation.dart

```diff
@@ -136,6 +143,7 @@ class _SideMenuAnimationState extends State<SideMenuAnimation>
 
   @override
   Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size.width;
```

Comment:
we don't need to get mediaquery here since we have `LayoutBuilder` and `constraints` below.